src/devices/tuya.ts

```diff
@@ -4229,22 +4229,41 @@ const definitions: Definition[] = [
         description: 'Wall-mount thermostat',
         fromZigbee: [tuya.fz.datapoints],
         toZigbee: [tuya.tz.datapoints],
-        onEvent: tuya.onEvent({timeStart: '1970'}),
+        onEvent: tuya.onEventSetLocalTime,
         configure: tuya.configureMagicPacket,
         exposes: [
-            e.climate().withSetpoint('current_heating_setpoint', 5, 60, 0.5, ea.STATE_SET)
+            tuya.exposes.frostProtection(),
+            e.battery(),
+            e.child_lock(),
+            e.text('schedule_weekday', ea.STATE_SET).withDescription('Workdays (6 times `hh:mm/cc.c`)'),
+            e.text('schedule_weekend', ea.STATE_SET).withDescription('weekend (2 times `hh:mm/cc.c)`'),
```

Comment:
What divisor should be used between the times?

Reply:
Values are space separated.
<code>Received MQTT message on 'zigbee2mqtt/zwt07/set' with data '{"schedule_weekday":"06:00/21.0°C 08:00/18.0°C 11:30/21.0°C 12:30/18.0°C 17:00/21.0°C 22:30/18.0°C"}'</code>
Note that this value is pre-populated in the GUI (or HA) with the current value of zwt07. Changes are more intuitive.